### PR TITLE
test: (e2e) skip state checks

### PR DESCRIPTION
**Pull request by AntonOlkhovskyi**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/5043
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: e2e/wdio/core/tests/object-number.e2e-spec.ts [hunk 1]
```diff
@@ -132,7 +132,7 @@ describe('object number test suite', function () {
     });
 
     describe('Check visual regression', function() {
-        xit('should check examples visual regression', () => {
+        it('should check examples visual regression', () => {
             objectNumberPage.saveExampleBaselineScreenshot();
             expect(objectNumberPage.compareWithBaseline()).toBeLessThan(3);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/object-number.e2e-spec.ts:137]: maybe increase to 5?

File: e2e/wdio/platform/tests/action-list-item.e2e-spec.ts [hunk 2]
```diff
@@ -44,8 +44,7 @@ describe('Action List Item Test Suite:', function() {
     });
 
     describe('Check visual regression', function() {
-        // TODO: enable after resolving https://github.com/SAP/fundamental-ngx/issues/4957
-        xit('should check examples visual regression', () => {
+        it('should check examples visual regression', () => {
             actionListPage.saveExampleBaselineScreenshot();
             expect(actionListPage.compareWithBaseline()).toBeLessThan(3);
```
Comment (Seamoo13) [inline e2e/wdio/platform/tests/action-list-item.e2e-spec.ts:49]: maybe increase to 5?

File: e2e/wdio/platform/tests/object-list-item.e2e-spec.ts [hunk 3]
```diff
@@ -124,7 +125,7 @@ describe('Object list item suite:', function() {
     });
 
     describe('Check visual regression', function() {
-        xit('should check examples visual regression', () => {
+        it('should check examples visual regression', () => {
             objListPage.saveExampleBaselineScreenshot();
             expect(objListPage.compareWithBaseline()).toBeLessThan(3);
```
Comment (Seamoo13) [inline e2e/wdio/platform/tests/object-list-item.e2e-spec.ts:130]: same, increase to 5

File: wdio-local.conf.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ exports.config = {
     // directory is where your package.json resides, so `wdio` will be called from there.
     //
     specs: [
-        './e2e/wdio/**/*.e2e-spec.ts'
+        './e2e/wdio/core/**/select.e2e-spec.ts'
```
Comment (Seamoo13) [inline wdio-local.conf.js:24]: need to change back to default value to run all tests.

**Review by Seamoo13 — CHANGES_REQUESTED** [review]
main thing is to revert config file back to run all tests.

**Review by nikvalor — APPROVED** [review]

**Review by Seamoo13 — APPROVED** [review]
Approved, but you need to mark the checkboxes in the PR description.

**Pull request closed without merging**